e2e/__snapshots__/templates.test.js.snap

```diff
@@ -3439,12 +3488,18 @@ instantsearchJsApp({
 or an \`HTMLElement\`:
 
 \`\`\`js
-instantsearchJsApp({
+{
+  {
+    camelCaseName;
```

Comment:
this seems wrong

Reply:
It's fixed and not there anymore.

```
{{ camelCaseName }}({
  container: '#{{ name }}',
  // ...
});
```

↑ this snippet in README.md was prettified at that moment, which caused that issue.